Dockerfile

```diff
@@ -1,15 +1,71 @@
-FROM php:8.1-fpm-alpine
+FROM ubuntu:22.04
 
-RUN apk add --no-cache bash \
-    && php -r "readfile('https://getcomposer.org/installer');" | php -- --install-dir=/usr/bin/ --filename=composer
+# Set timezone
+RUN \
+    echo "Asia/Dhaka" > /etc/timezone   && \
+    echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && \
+    ln -sf /usr/share/zoneinfo/EET /etc/localtime
 
-COPY . /var/www/html
-WORKDIR /var/www/html
+# Install basic packages
+RUN \
+    apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y \
+    software-properties-common \
+    apt-utils \
+    tzdata \
+    locales \
+    vim \
+    htop \
+    zip \
+    bzip2 \
+    wget \
+    curl \
+    git \
+    build-essential \
+    subversion \
+    poppler-utils \
+    sudo && \
+    apt-get autoremove -y && \
+    apt-get clean all
```

Comment:
This is just the random packages I chose. But if your application requires specific packages, you should add or remove them from here.